server/routers/index.js

```diff
@@ -2,8 +2,10 @@ const express = require('express');
 const router = express.Router();
 const deliverableRouter = require('./deliverables');
 const sessionsRouter = require('./sessions');
+const juriesRouter = require('./juries');
 
 router.use('/deliverable', deliverableRouter);
 router.use('/sessions', sessionsRouter);
+router.use('grades', juriesRouter);
```

Comment:
Either name this `juries` or change juriesRouter to grades router. Also don't forget the `/`